src/components/external-source/ExternalSourcePanelEntry.svelte

```diff
@@ -0,0 +1,140 @@
+<svelte:options immutable={true} />
+
+<script lang="ts">
+  import Eye from 'bootstrap-icons/icons/eye-fill.svg?component';
+  import EyeSlash from 'bootstrap-icons/icons/eye-slash.svg?component';
+  import { get } from 'svelte/store';
+  import { derivationGroupVisibilityMapWritable, externalSources } from '../../stores/external-source';
+  import { plan } from '../../stores/plan';
+  import { plugins } from '../../stores/plugins';
+  import type { User } from '../../types/app';
+  import type { DerivationGroup, ExternalSourceSlim } from '../../types/external-source';
+  import effects from '../../utilities/effects';
+  import { formatDate } from '../../utilities/time';
+  import { tooltip } from '../../utilities/tooltip';
+  import Collapse from '../Collapse.svelte';
+
+  export let derivationGroup: DerivationGroup;
+  export let user: User | null;
+
+  let relevantSources: ExternalSourceSlim[] = [];
+
+  $: enabled = $derivationGroupVisibilityMapWritable[derivationGroup.name] ?? true;
+  $: relevantSources = $externalSources.filter(source => derivationGroup.name === source.pkey.derivation_group_name);
+
+  function onChange() {
+    derivationGroupVisibilityMapWritable.set({
+      ...get(derivationGroupVisibilityMapWritable),
+      [derivationGroup.name]: !enabled,
+    });
+  }
+
+  async function deleteEmptyDerivationGroup() {
+    if (enabled) {
+      // Delete plan derivation group association
+      await effects.deleteDerivationGroupForPlan(derivationGroup.name, $plan, user);
+    }
+    // Delete the derivation group itself
+    await effects.deleteDerivationGroup(derivationGroup.name, user);
+  }
+</script>
+
+<div class="external-source-pairing">
+  <Collapse
+    title={derivationGroup.name}
+    tooltipContent={'Derivation group ' + derivationGroup.name}
+    defaultExpanded={false}
+  >
+    <span slot="right" style:display="flex">
+      <p class="derived-event-text">
+        {derivationGroup.derived_event_total} derived events
+      </p>
+      {#if enabled === true}
+        <button
+          class="st-button icon"
+          on:click|stopPropagation={onChange}
+          use:tooltip={{ content: 'Show in timeline', placement: 'top' }}
+        >
+          <Eye style="color: rgba(143, 143, 143, 1)" />
+        </button>
+      {:else}
+        <button
+          class="st-button icon"
+          on:click|stopPropagation={onChange}
+          use:tooltip={{ content: 'Hide in timeline', placement: 'top' }}
```

Comment:
Can you switch the hide/show language in these tooltips so that the tooltip reflects the action that would occur on click as opposed to current state?